Plugin.php

```diff
@@ -10,9 +10,21 @@ class Plugin implements PluginEntryPointInterface
     /** @return void */
     public function __invoke(RegistrationInterface $registration, SimpleXMLElement $config = null)
     {
-    	$registration->addStubFile(__DIR__ . '/stubs/Mockery.php');
-    	
-    	require_once __DIR__ . '/hooks/MockReturnTypeUpdater.php';
+        $registration->addStubFile(__DIR__ . '/stubs/Mockery.php');
+
+        if (class_exists('PHPUnit_Framework_TestCase')
+            || (class_exists('\PHPUnit\Runner\Version')
```

Comment:
These checks are almost verbatim from Mockery (https://github.com/mockery/mockery/blob/master/library/Mockery/Adapter/Phpunit/MockeryPHPUnitIntegration.php#L25) but I added the additional `class_exists` check on `\PHPUnit\Runner\Version` just in case somebody is using Mockery outside of PHPUnit.